rrmngmnt/network.py

```diff
@@ -500,9 +500,9 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
         :return: True/False
         :rtype: bool
         """
-        cmd = ["ping", dst, "-c", count, "-s", size]
-        if size != "1500":
-            cmd.extend(["-M", "do"])
+        cmd = ["ping", dst, "-c", count]
+        if size and size.isdigit():
```

Comment:
I would remove this condition, and use `str(size)` in following line. I don't like approach to ignore parameter when it has wrong form - it is always better to fail.